### PR TITLE
[examples] upgrade code for seed setting  

### PR DESCRIPTION
**Pull request by faaany**
## What does this PR do?
As the title suggests, this PR updates the code for seed setting in `cv_example.py`. The current code only applies for cuda device.

cc @SunMarc and @muellerzr

### CODE REVIEW

File: examples/cv_example.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 from torch.optim.lr_scheduler import OneCycleLR
 from torch.utils.data import DataLoader, Dataset
 from torchvision.transforms import Compose, RandomResizedCrop, Resize, ToTensor
+from transformers import set_seed
```
Comment (SunMarc) [inline examples/cv_example.py:25]: You can use set_seed from accelerate.utils instead !

**Review by SunMarc — APPROVED** [review]
Thanks ! Just a nit

File: examples/cv_example.py (hunk 1 above)
Comment (faaany) [reply examples/cv_example.py]: Sure, updated.
